mitosheet/mitosheet/api/api.py

```diff
@@ -51,13 +51,22 @@
 # View > Show Log Console > in the console set Log Level to Debug
 THREADED = True
 
-# JupyterLite does not support multiple threads, so we have to set it equal
-# to False in this case
-if is_jupyterlite():
-    THREADED = False
 
-if is_streamlit():
-    THREADED = False
+def get_api_should_be_threaded() -> bool:
+
+    # JupyterLite does not support multiple threads, so we have to set it equal
+    # to False in this case
```

Comment:
Update comment for streamlit and dash? Anything else important to know here? I think there are some important differences you called out between Streamlit and Dash, but maybe that should be documented elsewhere?